cluster/cluster_members.go

```diff
@@ -188,3 +208,19 @@ func GetClusterMemberAPIExtensions(ctx context.Context, tx *sql.Tx) ([]extension
 
 	return results, nil
 }
+
+// The cluster members table was renamed with updateFromV4, so we cannot be sure of its name prior to applying that update.
```

Comment:
The comment is the reason for adding this helper but isn't precisely describing it's behavior.